### DISCUSSION TITLE
Problem to simulate liquid sodium

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12965] [Q&A]
Dear developers,

I am planning to use FDS to simulate the combustion of liquid sodium at several hundred degrees Celsius. I have referred to the liquid pyrolysis and methanol evaporation cases, but I still don't have any clue at this point, could you please give some guidance? Thanks!

Kind regards,

LI

**Comment by rmcdermo** [comment]
You first need to read the user guide chapter on fire and pyrolysis.  Then please ask specific questions about what you do not understand.  Just asking us to give "guidance" is too nebulous.  If you have no idea how to use FDS, then starting out with a complex research problem is not a good idea.
